internal/service/advancedclustertpf/resource_test_cases_test.go

```diff
@@ -318,3 +325,153 @@ func checksTenantUpgraded(projectID, name string) resource.TestCheckFunc {
 	checks = acc.AddAttrChecks(resourceName, checks, checkMap)
 	return resource.ComposeAggregateTestCheckFunc(originalChecks, resource.ComposeAggregateTestCheckFunc(checks...))
 }
+
+func ReplicasetAdvConfigUpdate(t *testing.T) *resource.TestCase {
```

Comment:
is it exported because it will be used in both sdkv2 and tpf packages?

Reply:
Good eye. To reuse we would have to put it into a different package as test packages cannot be imported.
For now, I think these test cases will only be run for TPF, otherwise I would have to re-write config to be compatible with SDKv2

Reply:
Renamed in 9fc4d8be039bb5d12205dcc12d3ff906444013f3